packages/fluttium/lib/src/actions/expect_not_visible.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:fluttium/fluttium.dart';
+
+/// {@template expect_not_visible}
+/// Asserts that a node that matches the arguments is not visible.
+/// {@endtemplate}
+class ExpectNotVisible extends Action {
+  /// {@macro expect_not_visible}
+  const ExpectNotVisible({
+    required this.text,
+    this.timeout = 500,
```

Comment:
Curious why there is a default timeout for `ExpectNotVisible` but not `ExpectVisible`?

Reply:
Because the `ExpectNotVisible` should fail faster, as you expect it to not be visible potentially yet

Reply:
hmm okay I see your point. I don't think it's a big deal but fwiw I assumed that ExpectNotVisible would reuse the ExpectVisible implementation and negate the result. I'm totally with leaving it as is though.

Reply:
Good point, extending from that with a custom super timeout value!